docs/general/installation/truenas.md

```diff
@@ -0,0 +1,246 @@
+
+Mounting an SMB share allows data synchronization between the share and the app.
+The SMB share mount does not include ACL protections at this time. Permissions are currently limited to the permissions of the user that mounted the share. Alternate data streams (metadata), finder colors tags, previews, resource forks, and MacOS metadata is stripped from the share along with filesystem permissions, but this functionality is undergoing active development and implementation planned for a future TrueNAS SCALE release.
+
+For all types, enter a **Mount Path** to be used within the Jellyfin container.
+For example, the local **Host Path** /mnt/tank/video/movies could be assigned the **Mount Path** /media/movies.
+
+<details>
+  <summary>Additional Storage Fields</summary>
+
+  <table>
+    <thead>
+      <tr>
+        <th>Type</th>
+        <th>Field</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>All</td>
+        <td>Mount Path</td>
+        <td>The virtual path to mount the storage within the container.</td>
+      </tr>
+      <tr>
+        <td>Host Path</td>
+        <td>Host Path</td>
+        <td>The local path to an existing dataset on the System.</td>
+      </tr>
+      <tr>
+        <td>ixVolume</td>
+        <td>Dataset Name</td>
+        <td>The name for the dataset the system creates.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Server</td>
+        <td>The server for the SMB share.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Share</td>
+        <td>The name of the share.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Domain (Optional)</td>
+        <td>The domain for the SMB share.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Username</td>
+        <td>The user name used to access the SMB share.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Password</td>
+        <td>The password for the SMB share user.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Size (in Gi)</td>
+        <td>The quota size for the share volume. You can edit the size after deploying the application if you need to increase the storage volume capacity for the share.</td>
+      </tr>
+    </tbody>
+  </table>
+</details>
+
+### Resource Configuration Settings
+
+Accept the default values in **Resources Configuration** or enter new CPU and memory values
+By default, this application is limited to use no more than 4 CPU cores and 8 gibibytes available memory.
+
+![Resource Limits](/images/docs/install-truenas-12.png)
+
+To customize the CPU and memory allocated to the container Jellyfin uses, enter new CPU values as a plain integer value followed by the suffix m (milli).
+Default is 4000m.
+
+Accept the default value 8Gi allocated memory or enter a new limit in bytes.
+Enter a plain integer followed by the measurement suffix, for example 129M or 123Mi.
```

Comment:
I'd think it would be better to give examples in the GB range. It's not really recommended to allocate less than 4GB of RAM for JF